variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "prefix" {
+    description = "The prefix used for all resources in this environment"
+    default = "todo"
+}
+variable "location" {
+    description = "The Azure location where all resources in this deployment should be created"
+    default = "uksouth"
+}
+
+variable "resource_group" {
+    default = "Capita1_LouisEgbufor_ProjectExercise"
+}
+
+variable "GITHUB_CLIENT_ID" { 
+}
+
+variable "GITHUB_CLIENT_SECRET" { 
```

Comment:
Like the output variables, we [can mark these as sensitive](https://www.terraform.io/docs/language/values/variables.html#suppressing-values-in-cli-output) to reduce the risk of having them be printed to the console logs - we should apply that to any sensitive variables